Dockerfile

```diff
@@ -28,6 +28,8 @@ RUN easy-add --var version=0.1.1 --var app=set-property --file {{.app}} --from h
 
 RUN easy-add --var version=1.2.0 --var app=restify --file {{.app}} --from https://github.com/itzg/{{.app}}/releases/download/{{.version}}/{{.app}}_{{.version}}_linux_${ARCH}.tar.gz
 
+RUN easy-add --var version=0.5.0 --var app=mc-monitor --file {{.app}} --from https://github.com/itzg/{{.app}}/releases/download/{{.version}}/{{.app}}_{{.version}}_linux_${ARCH}.tar.gz
```

Comment:
Much better, thanks

Reply:
Indeed.  I can't believe I didn't notice that before I spent the time doing a multi-stage build :).